src/cli/scripts/migrateContentServer.ts

```diff
@@ -130,8 +130,14 @@ export async function migrateContentServer({
   );
 
   const servers: {
-    [serverKey: string]: { dataModel: DataModel<"raw">; url: string };
-  } = await fs.readJSON(serversPath);
+    [serverKey: string]: {
+      dataModel: DataModel<"raw">;
+      url: string;
+    };
+  } & { contentServerVersion?: string } = await fs.readJSON(serversPath);
+
+  const contentServerVersion = servers.contentServerVersion;
+  delete servers.contentServerVersion;
```

Comment:
Because everywhere it's used, the expected structure of `servers` is `{ [serverKey: string]: { ... } }`, and a lot of `_mapValues`, `_findKey`, etc. methods are applied to it lower in the migration tree. Not deleting the `contentServerVersion` part would break all those

Reply:
Could you add a comment above explaining this?